### DISCUSSION TITLE
How to disable coloring in an app (not SBT)

### DISCUSSION THREAD

**Discussion by joan38** [outr/scribe#492] [Q&A] [answered]
Hi,
Using scribe, I endup with logs like:
```
[36m[1m2023.07.19 11:04:05:289[0m[36m[0m [3mio-compute-2[0m [34mINFO[0m ........
```
any idea how I can disable the coloring?

**Comment by darkfrog26** [comment, marked as answer]
The easiest way is:
```scala
scribe.output.format.OutputFormat.default = scribe.output.format.ASCIIOutputFormat
```
You can also set the environment value `SCRIBE_OUTPUT_FORMAT` to "ASCII". Scribe attempts to auto-detect if ANSI is supported. Can you give me more information about your set up and why ANSI isn't supported?

**Comment by joan38** [comment]
That's perfect thanks @darkfrog26 !

We are running a Spark job on a cluster that somehow (Netflix's platform does it) dumps the stdout and stderr into files.
I'm not sure how scribe could figure out if ANSI is or isn't supported?

**Comment by darkfrog26** [comment]
On JVM I generally just look to see if the `TERM` environment variable is set, which it apparently is for you.
